### PR TITLE
Mobile fixes, small style updates

### PR DESCRIPTION
**Pull request by argyleink**
image

made a few other small design changes, tried to not change much

### CODE REVIEW

**Review by MylesBorins — APPROVED** [review]
tested locally LGTM

**Comment by argyleink** [comment]
image

working across firefox, safari, opera, chrome

File: scripts/vnu-jar.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    'Property "stroke-color" doesn\'t exist.'
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
    '"stroke-color": Property "stroke-color" doesn\'t exist.'
``

Comment (argyleink) [reply scripts/vnu-jar.js]: will try that next 👍

File: scripts/vnu-jar.js [hunk 2]
```diff
@@ -22,7 +22,8 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    '"stroke-color" doesn\'t exist.'
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
    '"stroke-color": Property "stroke-color" doesn\'t exist.'
``

File: scripts/vnu-jar.js [hunk 3]
```diff
@@ -22,7 +22,9 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    '"stroke-color": Property "stroke-color" doesn\'t exist.'
+``
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
```

File: scripts/vnu-jar.js [hunk 4]
```diff
@@ -22,7 +22,8 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    '"stroke-color": Property "stroke-color" doesn\'t exist.'
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
    '*stroke-color*'
```

File: scripts/vnu-jar.js [hunk 5]
```diff
@@ -22,7 +22,8 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    'stroke-color'
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
    '*stroke-color*'
```

File: scripts/vnu-jar.js [hunk 6]
```diff
@@ -22,7 +22,8 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    '*stroke-color*'
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
    'stroke-color*'
```

File: layouts/black-lives-matter.hbs [hunk 7]
```diff
@@ -81,12 +96,41 @@
...
+        transform: translateX(-110%);
+      }
+
+      section {
+        margin: 0;
+      }
+    }
+
+    @media not all and (min-resolution: 0.001dpcm) {
+      @supports (-webkit-appearance: none) and (stroke-color: transparent) {
```
Comment (MylesBorins) [inline layouts/black-lives-matter.hbs]:
```suggestion
      @supports (-webkit-appearance: none) and (-webkit-text-stroke-color: transparent) {
```

This doesn't appear to work right now BUT it does pass the linter

File: scripts/vnu-jar.js [hunk 8]
```diff
@@ -22,7 +22,8 @@ childProcess.exec('java -version', (error, stdout, stderr) => {
     'Text run is not in Unicode Normalization Form C.',
     // These happen due to the commented out English HTML code some translations have...
     // They should be removed at some point
-    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
+    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.',
+    'stroke-color*'
```
Comment (MylesBorins) [inline scripts/vnu-jar.js]:
```suggestion
    'The document is not mappable to XML 1.0 due to two consecutive hyphens in a comment.'
```

**Pull request merged**